modules/apps/site-initializer/site-initializer-team-extranet/src/main/zippableResources/fragments/team-extranet/fragments/main-navbar/index.css

```diff
@@ -41,29 +86,38 @@
 	margin-bottom: 0;
 }
 
-.main-navbar .btn-close,
-.main-navbar.open .btn-menu {
-	display: none;
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+textarea:-webkit-autofill,
+textarea:-webkit-autofill:hover,
+textarea:-webkit-autofill:focus,
+select:-webkit-autofill,
+select:-webkit-autofill:hover,
+select:-webkit-autofill:focus {
+	-webkit-text-fill-color: var(--white);
+	-webkit-box-shadow: 0 0 0px 1000px transparent inset;
+	box-shadow: 0 0 0px 1000px transparent inset;
+	color: var(--white);
+	transition: background-color 5000s ease-in-out 0s;
 }
```

Comment:
We could probably remove as much specificity by removing for example `.main-navbar` in those 5 styles